.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    name: build
+    strategy:
+      matrix:
+        go-version: [1.16, 1.17, 1.18]
+        os: [ubuntu-latest] #, macos-latest, windows-latest]
```

Comment:
Assume the comments are leftovers from testing locally?

Reply:
fixed